src/deepsparse/server/config.py

```diff
@@ -49,6 +49,63 @@ class ImageSizesConfig(BaseModel):
     )
 
 
+class SystemLoggingGroup(BaseModel):
+    """
+    Holds the configuration for a single system logging group.
+    """
+
+    enable: bool = Field(
+        default=False, description="Whether to enable the system logging group"
+    )
+
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of target loggers to log to. "
+        "If None, logs to all the available loggers",
+    )
```

Comment:
i guess someone could disable by setting target_loggers to empty list right? think its easier to understand how to configure by removing enable field?
```suggestion
    target_loggers: Optional[List[str]] = Field(
        default=None,
        description="The list of target loggers to log to. "
        "If None, logs to all the available loggers",
    )
```

Reply:
I'm in favor of having the explicit enable field to remove any ambiguity between defaulting to all loggers and `None` vs empty list

Reply:
I like it